src/main/webapp/about.jsp

```diff
@@ -64,6 +64,15 @@
         This page should also be used to describe the features and improvements
         you've added.
       </p>
```

Comment:
I think we can delete the boilerplate from the initial CodeU starter code. What does everyone else think?

Reply:
I think that's a good idea!

Reply:
You should be able to add another commit to this branch, then push it again. That way the changes will show here in the pull request. :)